site/js/modules/webui/course.js

```diff
@@ -1,10 +1,14 @@
-import * as Render from './render.js'
-import * as Routing from './routing.js'
+import * as Autograder from '../autograder/base.js';
+import * as Render from './render.js';
+import * as Routing from './routing.js';
+
+const recipientDocsLink = "https://github.com/edulinq/autograder-server/blob/main/docs/types.md#course-user-reference-courseuserreference";
```

Comment:
This is not CONSTANT_CASE:
https://en.wikipedia.org/wiki/Naming_convention_(programming)#Examples_of_multiple-word_identifier_formats